File: macVisual1/macVisual1/ViewController.m
#import "ViewController.h"

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    [_lbMensaje setStringValue:@"Hola a todos"];
    // Do any additional setup after loading the view.
}


- (void)setRepresentedObject:(id)representedObject {
    [super setRepresentedObject:representedObject];

    // Update the view, if already loaded.
}

- (IBAction)clickEnBoton:(id)sender {
    if ([sender isKindOfClass:[NSButton class]]) {
        NSString *mensaje = [NSString stringWithFormat:@"Mi nombre es %@ y tengo %@ años.", _txtNombre.stringValue, _txtEdad.stringValue];
        
        [_lbMensaje setStringValue:mensaje];
        
        NSLog(@"Click en un botón");
    } else {
        NSLog(@"Click en otra cosa");
    }
}
@end

File: swift1/swift1/ViewController.swift
import Cocoa

class ViewController: NSViewController {
    
    func saludar() {
        print("Hola")
    }
    func saludarConParametros(nombre: String) {
        print ("Hola \(nombre)")
    }
    
    func obtenerSaludo(nombre: String, apellido: String) -> String {
        return "Hola \(nombre) \(apellido)"
    }

    override func viewDidLoad() {
        super.viewDidLoad()

        // Do any additional setup after loading the view.
    }

    override var representedObject: Any? {
        didSet {
        // Update the view, if already loaded.
        }
    }
}
